src/Transports.Subscriptions.WebSockets/WebSocketConnectionFactory.cs

```diff
@@ -14,26 +15,38 @@ public class WebSocketConnectionFactory<TSchema> : IWebSocketConnectionFactory<T
         private readonly IGraphQLExecuter<TSchema> _executer;
         private readonly IEnumerable<IOperationMessageListener> _messageListeners;
         private readonly IDocumentWriter _documentWriter;
+        private readonly IServiceScopeFactory _serviceScopeFactory;
 
         public WebSocketConnectionFactory(ILogger<WebSocketConnectionFactory<TSchema>> logger,
             ILoggerFactory loggerFactory,
             IGraphQLExecuter<TSchema> executer,
             IEnumerable<IOperationMessageListener> messageListeners,
             IDocumentWriter documentWriter)
+            : this(logger, loggerFactory, executer, messageListeners, documentWriter, null)
+        {
+        }
+
+        public WebSocketConnectionFactory(ILogger<WebSocketConnectionFactory<TSchema>> logger,
+            ILoggerFactory loggerFactory,
```

Comment:
```suggestion
        public WebSocketConnectionFactory(
            ILogger<WebSocketConnectionFactory<TSchema>> logger,
            ILoggerFactory loggerFactory,
```

The same for the ctor above.